src/main/java/seedu/superta/logic/commands/GradeAssignmentCommand.java

```diff
@@ -19,15 +23,15 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Grades an assignment.\n"
         + "Parameters: "
-        + "tg/TUTORIAL-GROUP-ID "
-        + "as/ASSIGNMENT-NAME "
-        + "st/STUDENT-ID "
-        + "m/MARKS\n"
+        + PREFIX_TUTORIAL_GROUP_ID + "TUTORIAL-GROUP-ID "
```

Comment:
Should be PREFIX_GENERAL_TUTORIAL_GROUP_ID if I'm not wrong.

Reply:
Thanks. Will change it.